test/rapids_pre_commit_hooks/test_alpha_spec.py

```diff
@@ -12,15 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import contextlib
+import os.path
 from itertools import chain
 from textwrap import dedent
 from unittest.mock import MagicMock, Mock, call, patch
 
 import pytest
 import yaml
+from packaging.version import Version
+from rapids_metadata import all_metadata
 
 from rapids_pre_commit_hooks import alpha_spec, lint
 
+latest_version, latest_metadata = max(
+    all_metadata.versions.items(), key=lambda item: Version(item[0])
+)
```

Comment:
This is fine for now (don't block the PR on this comment), but I think we should consider just adding an attribute like `.latest_version` on `rapids_metadata.RAPIDSMetadata` for this purpose, so code wanting to get this information doesn't need to be coupled to the implementation details of that class, like the fact that `.versions` is a dictionary keyed by version.

Reply:
I don't disagree with the idea of adding `.latest_version` (and I'll probably do it), but as far as I'm concerned, `.versions` is not an implementation detail, but part of the public API. Knowing that it's a string-keyed dict is fine with me.

Reply:
Ok fair enough, I'm ok with that.